pkg/cache/placeholder_manager_test.go

```diff
@@ -173,3 +173,26 @@ func TestCleanUp(t *testing.T) {
 	assert.Equal(t, exist, false)
 	assert.Equal(t, len(placeholderMgr.orphanPod), 0)
 }
+
```

Comment:
could u pls add a test case to test the start/stop function?

Reply:
done.